data_steward/cdr_cleaner/clean_cdr.py

```diff
@@ -133,6 +132,7 @@
 
 EHR_CLEANING_CLASSES = [
     (CleanMappingExtTables,),  # should be one of the last cleaning rules run
+    (RemoveInvalidProcedureSourceRecords,),
```

Comment:
Is this the right place to put this cleaning rule? The ticket mentions `it needs to run at the clean step,` and I think `the clean step` with the old terminology can mean the combined stage. Let me know if you have already had some conversations about it.

Reply:
@lrwb-aou Should this be in the unioned cleaning or combined cleaning?

Reply:
@lledezma and @hiro-mishima After reviewing the comments in DC-583, I think this rule is expected to run at `UNIONED_EHR_CLEANING_CLASSES`.  

The cleaning engine was a completely different piece of code when this rule was originally written and being updated when the ticket to update the rule was written.